frontend/package.json

```diff
@@ -7,7 +7,8 @@
     "reason:clean": "bsb -clean-world",
     "webpack:dev": "webpack-dev-server",
     "webpack:build": "NODE_ENV=production webpack",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "fmt": "refmt --in-place **/*.re"
```

Comment:
I can rename this to `yarn format` if you prefer over `yarn fmt`.

Reply:
Looks good! Merged.